pkg/sabactl/cmd/root.go

```diff
@@ -102,4 +115,6 @@ func Execute() {
 
 func init() {
 	rootCmd.PersistentFlags().StringVar(&flagServer, "server", "http://localhost:10080", "<Listen IP>:<Port number>")
+	rootCmd.PersistentFlags().StringVar(&flagTLSServer, "tls-server", "https://localhost:10443", "<Listen IP>:<Port number>")
+	rootCmd.PersistentFlags().BoolVar(&flagInsecure, "insecure", false, "Disable TLS verification")
```

Comment:
Do we always use the `--insecure` option to run `sabactl crypts delete`?
Don't we need a way to load additional CA files?

Reply:
In normal, we don't need to use`--insecure=true` because we only use sabactl in bootserver, and rootCA cert is installed in the boot server's trust store.